broker-cli/proto/broker.proto

```diff
@@ -69,50 +69,27 @@ enum Side {
   ASK = 1;
 }
 
-/**
- * Statuses of an engine
- */
-enum EngineStatuses {
-  // Default state of the engine
-  UNKNOWN = 0;
-  // Engine is unavailable
-  UNAVAILABLE = 1;
-  // Wallet does not exist
-  NEEDS_WALLET = 2;
-  // Wallet exists but is not locked
-  LOCKED = 3;
-  // Wallet is unlocked but the configuration of the node and chain sync status are not known
-  UNLOCKED = 4;
-  // Engine is not yet synced to chain
-  NOT_SYNCED = 5;
-  // Engine is validated and ready for use
-  VALIDATED = 6;
-}
-
-/**
- * Statuses of the relayer
- */
-enum RelayerStatuses {
-  // Relayer is available
-  OK = 0;
-  // Relayer is not available
-  UNAVAILABLE = 1;
-}
-
-/**
- * Statuses of an orderbook
- */
-enum OrderbookStatuses {
-  // Orderbook is synced
-  OK = 0;
-  // Orderbook is not synced
-  NOT_SYNCED = 1;
-}
 
 /**
  * HealthCheckResponse provides information on the health of a BrokerDaemon.
  */
 message HealthCheckResponse {
+  // Statuses of an orderbook
+  enum OrderbookStatuses {
+    // Orderbook is synced
+    OK = 0;
+    // Orderbook is not synced
+    NOT_SYNCED = 1;
+  }
+
+  // Statuses of the relayer
+  enum RelayerStatuses {
+    // Relayer is available
+    RELAYER_OK = 0;
```

Comment:
Maybe we should prefix all of them? e.g. `RELAYER_UNAVAILABLE` and `ORDERBOOK_OK` ?

Reply:
This makes sense since they are status codes that we are setting. ill make the change